Pre-existing projects have a `description`

aminemboulouma opened the pull request:
Fixed #197

jankeromnes commented on lib/machines.js:
```diff
@@ -476,6 +476,9 @@ function getProject (projectId, create) {
     return null;
   }
 
+  if(!project.description)  {
```
Nit: Please explain this code by adding a comment above this line, e.g. `// Temporary migration code: Previous projects didn't have a description.`.

jankeromnes replied:
Nit: Our coding style normally has a space between `if` and `(`. Please add one.

jankeromnes commented on lib/machines.js:
```diff
@@ -476,6 +476,9 @@ function getProject (projectId, create) {
   }
 
+  if(!project.description)  {
+    project.description='';
```
Nit: Our coding style normally has spaces before and after `=`, please add them.

jankeromnes commented on lib/machines.js:
```diff
@@ -476,6 +476,9 @@ function getProject (projectId, create) {
 
+  if(!project.description)  {
+    project.description='';
+  }
```
Nit: Please add an empty line below this `}` in order to separate the two code blocks.

aminemboulouma commented:
Great to know ! I have updated my code with all the nits and added the comment.
Great to know how you are working on the project, thank you for sharing  that with me !

jankeromnes commented on lib/machines.js:
```diff
@@ -476,6 +476,11 @@ function getProject (projectId, create) {
     return null;
   }
 
+// Temporary migration code: Previous projects didn't have a description.
```
Nit: Please indent this line with 2 spaces.

jankeromnes commented on lib/machines.js:
```diff
@@ -476,6 +476,11 @@ function getProject (projectId, create) {
   }
 
+// Temporary migration code: Previous projects didn't have a description.
+  if (!project.description)  {
```
Nit: Please remove one of the two spaces before `{`.

aminemboulouma commented:
I think it is okay now, what do you think about it ?

jankeromnes commented:
> I think it is okay now, what do you think about it ?

Thanks! You fixed the double space issue. However Travis still found two more issues:

> /home/travis/build/JanitorTechnology/janitor/lib/machines.js
>   479:1  error  Expected indentation of 2 spaces but found 0  indent
>   483:1  error  Trailing spaces not allowed                   no-trailing-spaces

Note that you can simply run `npm run lint-fix` in the IDE to automatically fix all these issues. 😄 (Sorry, I should have mentioned this command earlier!)

aminemboulouma commented:
It is okay, I have done so, is there anything I forgot about ?

jankeromnes commented:
All green! Ready for launch 🚀✔️

jankeromnes commented:
Merged! 🏁 Thanks again!
